jumpscale/packages/vdc_dashboard/bottle/api/deployments.py

```diff
@@ -247,6 +247,38 @@ def threebot_vdc():
     )
 
 
+@app.route("/api/vdc/expiration", method="GET")
+@package_authorized("vdc_dashboard")
+def vdc_expiration():
+    vdc = get_vdc()
+    data = get_expiration_data()
+    return HTTPResponse(j.data.serializers.json.dumps(data), status=200, headers={"Content-Type": "application/json"})
+
+
+@app.route("/api/vdc/plan/price", method="GET")
+@package_authorized("vdc_dashboard")
+def vdc_expiration():
+    vdc = get_vdc()
+    data = {"price": math.ceil(vdc.calculate_spec_price())}
+    return HTTPResponse(j.data.serializers.json.dumps(data), status=200, headers={"Content-Type": "application/json"})
+
+
+@app.route("/api/vdc/plan/autoscalable", method="GET")
+@package_authorized("vdc_dashboard")
+def vdc_plan():
+    vdc = get_vdc()
+    data = {"autoscalable": check_plan_autoscalable()}
```

Comment:
`vdc` is not used here? or do we need to call `get_vdc` to fetch information that  `check_plan_autoscalable` depends on?

Reply:
No, we don't need it.